src/User/Tests/Infra/Security/SecurityUserCheckerTest.php

```diff
@@ -16,28 +16,26 @@
 
 final class SecurityUserCheckerTest extends TestCase
```

Comment:
you can rebase btw.. this test is now gone :)

Reply:
Yup, I did about an hour or so ago after your last PR went in (a few new things popped up on the PHPStan tests)